modules/PhenoDigm.py

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/env python3
+"""Evidence parser for the animal model sources from PhenoDigm."""
+
+import argparse
+import logging
```

Comment:
I'm a bit confused, because as far as I can see my implementation already follows this convention: https://github.com/opentargets/evidence_datasource_parsers/pull/80/files#diff-2b946d8411b4433f2c4e427134659d102319c5317c1981bccc25d4972c32c9c2R323

The only thing it lacks is the else block, but this is only because the `stderr` stream handler is [enabled by default](https://docs.python.org/3/library/logging.handlers.html#logging.StreamHandler) and doesn't need to be specified.